src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -198,20 +198,22 @@ export const roomEditSchema = Yup.object().shape<RoomSchemaShape>({
   image_url: roomImageUrlSchema,
 });
 
+// @debt I'm pretty sure every one of these .from that have the same fromKey / toKey are redundant noops and should be removed
 export const venueEditSchema = Yup.object()
   .shape<Partial<SchemaShape>>({})
   .from("subtitle", "subtitle")
   .from("config.landingPageConfig.description", "description");
 
 // this is used to transform the api data to conform to the yup schema
+// @debt I'm pretty sure every one of these .from that have the same fromKey / toKey are redundant noops and should be removed
 export const editVenueCastSchema = Yup.object()
   .shape<Partial<VenueInput>>({})
   // possible locations for the subtitle
   .from("subtitle", "subtitle")
   .from("config.landingPageConfig.subtitle", "subtitle")
 
   .from("config.landingPageConfig.description", "description")
-  .from("profile_questions", "profileQuestions")
+  .from("profile_questions", "profile_questions")
```

Comment:
I'm okey with this for now, but we should get rid of the underscore syntax because the rest of our fields are using camelCase and it's confusing.

Reply:
Eventually sure, but the entire app uses this in the `_` form, and nowhere in the app references it in the `profileQuestions` form. So it will basically need to be a refactor of the entire app to do so.